control_toolbox/include/control_toolbox/pid_ros.hpp

```diff
@@ -184,6 +220,37 @@ class PidROS
    */
   void set_gains(double p, double i, double d, double i_max, double i_min, bool antiwindup = false);
 
+    /*!
+   * \brief Initialize the PID controller and set the parameters
+   * \param p The proportional gain.
+   * \param i The integral gain.
+   * \param d The derivative gain.
+   * \param i_max The max integral windup.
+   * \param i_min The min integral windup.
+   * \param u_max Upper output clamp.
+   * \param u_min Lower output clamp.
+   * \param trk_tc Specifies the tracking time constant for the 'back_calculation'
+        and 'conditioning_technique' strategies. If set to 0.0 when one of these
+        strategies is selected, a recommended default value will be applied.
```

Comment:
What's the recommended value?

Reply:
I also realized that later in the review